andela/titan-dev-ah-frontend
# #165716231 Fix Reactions Buttons on Scroll

#### What does this PR do?
- Create styles for fixing reactions buttons on viewing article
#### Description of Task to be completed?
- Adjust styles of view article
- Add reporting form on view article
#### How should this be manually tested?
- Clone the repos and check out the branch `ch-fix-reactions-buttons-on-scroll-165716231`
- Run `yarn` command to install all dependencies
- Run `yarn test` to run all tests. You should see all the tests passing. 
- start your backend server and front-end server by `yarn start`
- Create articles in your backend. 
- Grab a slug of one article. 
- Open your browser and navigate to `/articles/:slug`. You should see the article. When you scroll down, reactions buttons should stay visible. When you click on more button, you should see the report article. When you click on the report article, the report article form should appear. When you click cancel on report article form, the reporting form should disappear. 

#### What are the relevant pivotal tracker stories?

#165716231
#### Relevant screenshot
<img width="1368" alt="Screen Shot 2019-05-02 at 16 20 19" src="https://user-images.githubusercontent.com/34691070/57082100-62643600-6cf6-11e9-9bfd-82175e6bf848.png">
